src/webportal/src/app/job-submission/components/tools/job-ssh.jsx

```diff
@@ -130,6 +134,18 @@ export const JobSSH = ({
     [_onChangeExtras],
   );
 
+  const [sshGenerator, setSshGenerator] = useState({ isOpen: false });
```

Comment:
If this state has only one property, it's better to change it to boolean type for performance

Reply:
Done. Use bits defined on job-ssh page.

Reply:
I did not see ```bits``` changed anywhere, could we just use it as a const.

Reply:
Ok